packages/trader/src/Stores/Modules/Trading/trade-store.ts

```diff
@@ -267,6 +267,7 @@ export default class TradeStore extends BaseStore {
         last_tick_epoch?: number;
     } = {};
     tick_size_barrier = 0;
```

Comment:
@henry-deriv i'm afraid we don't use tick_size_barrier anywhere any more. can we remove all occurrences please 🙏🏻?